### ISSUE TITLE
Questions about training on my own dataset

### ISSUE THREAD

**Issue by supgy** [qinliuliuqin/Medical-Detection3d-Toolkit#9]
Hi author, Thank you for publicly disclosing your contribution.When I was training on my own dataset, I encountered some problems.

1. I trained the model for 2000 epochs, during testing, I found that some of the landmarks in the test samples could not be recognized which the predicted coordinates are [-1, -1, -1].(every sample have 25 landmarks)
<img width="554" height="729" alt="Image" src="https://github.com/user-attachments/assets/454ea69d-ba1e-4b58-90b9-f264071b4baa" />

2. Based on the above issue I guess if the crop_spacing casused this problem, so I changed crop_spacing from [2,2,2] to [1,1,1], but after training, in every sample not a single landmark can be recognized.

I dont konw what's the problem， can you give me some advice? Thanks a lot.

about my database: I didn't do any preprocessing, 
img:

<img width="318" height="603" alt="Image" src="https://github.com/user-attachments/assets/2db53281-f11f-4e05-bf86-69d08a8018f3" />

label:

<img width="159" height="348" alt="Image" src="https://github.com/user-attachments/assets/02a54fe5-b3fe-4c8e-b165-63c975bf4a50" />

<img width="999" height="879" alt="Image" src="https://github.com/user-attachments/assets/692dc49b-ae05-4627-bf39-b22f95fe9699" />

**Comment by qinliuliuqin** [comment]
Hi @supgy, thanks for your question. ```[-1, -1, -1]``` means the landmark is not detected. If each sample has 25 landmarks, then the prediction is false negative. To reduce false negative, you may need to consider the following suggestions:
1. Identify which landmarks are incorrectly predicted as false negatives. Based on the provided landmark mask, it seems these missed landmarks are likely very close to other landmarks, making them difficult for the model to differentiate. If this is the case, you may need to consider: 1) increase the resolution (which you had done), and 2) apply coarse-to-fine mechanism by training an additional fine-resolution model to predict those missed landmarks.
2. Increase training data size & apply advanced data augmentation.

Let me know if you have any further questions.
